### ISSUE TITLE
NVDA does not announces incoming messages at runtime in skype

### ISSUE THREAD

**Issue by lallerke1** [nvaccess/nvda#8810]
Steps to reproduce:
1. Open skype.
2. Write a message, and press enter.
3. NVDA does not announces written message in skype 8, as in skype 7.
4. Wait, stil a message appears from the other party.
5. NVDA does not read the incoming message, only by reviewing the window with html navigation keys.

Actual behavior:
NVDA does not read written, and incoming messages.

Expected behaviour:
NVDA shall read in runtime the incoming, and the sent messages.

System configuration:
Hungarian display language, but tested in english as well.

NVDA installed
alpha-16123,142e542a
But tested also in nvda 2018.3.2
Windows version:
windows 10 version 1308 build 17134.320

Other programms:
skype 8.31

The issue still exists after pc restart.

Yes, I have, nvda latest 2018.3.2

**Comment by dnz3d4c** [comment]
I think it is a duplicate of #8591.

**Comment by Adriani90** [comment]
As @k-kolev1985 reports in #8591, incomming messages are being reported correctly. But there seems to be a problem with outgoing messages. As this is covered in the referenced issue, I will close this as duplicate. Thanks.
